CI_JobHistory.py

```diff
@@ -241,9 +248,11 @@ def main():
     parser = argparse.ArgumentParser(description='Get the job history')
     parser.add_argument('--zone', help='specify the lease/zone', type= lambda arg:arg.split(','))
     parser.add_argument('--job_type', default='p', choices=['p','z','pa'], help= 'Specify the CI job type (Power(p) or s390x(z) or Power Auxillary(pa)), default is p')
+    parser.add_argument('--job_platform',default=' ', help='enter job_platform as heavy libvirt powervs upgrade')
```

Comment:
```suggestion
    parser.add_argument('--job_platform',default=' ', help='Specify the job_platform to fetch jobs, supported values are heavy / libvirt / powervs / upgrade')
```

Reply:
Instead of heavy, Can we add heavy build?

Reply:
ok

Reply:
**pass job_platform name using " " as below;-**
```
PS C:\Users\AlokGoswami\project\ci-monitoring-automation> python3 .\CI_JobHistory.py --job_platform "heavy build","libvirt"
['heavy build', 'libvirt']
1  4.12 libvirt
2  4.12 heavy build
3  4.13 libvirt
4  4.13 heavy build
5  4.14 libvirt
6  4.14 heavy build
7  4.15 libvirt
8  4.15 heavy build
9  4.16 libvirt
10  4.16 heavy build
11  4.17 libvirt
12  4.17 libvirt multi
13  4.17 heavy build multi
14  4.17 heavy build
15  All the above
Select the required ci's serial number with a space 1 2 3 4 5 6 
Enter Before date (YYYY-MM-DD): 2024-10-24
Enter After date (YYYY-MM-DD): 2024-10-24
Please select one of the option from Job History functionalities: 
1. Check Node Crash
2. Brief Job information
3. Detailed Job information
4. Failed testcases
5. Get builds with testcase failure
6. Get testcase failure frequency
Enter the option: 3
Checking runs from 2024-10-24 to 2024-10-24
--------------------------------------------------------------------------------------------------
4.13 libvirt
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-remote-libvirt-ppc64le/1849496263804850176
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.13.0-0.nightly-ppc64le-2024-10-22-143107
Lease Quota- libvirt-ppc64le-1-1
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.14 libvirt
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-remote-libvirt-ppc64le/1849511632544206848
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.14.0-0.nightly-ppc64le-2024-10-24-130542
Lease Quota- libvirt-ppc64le-2-0
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------```